config.sh

```diff
@@ -354,6 +354,12 @@ if test "${OS}" = "emscripten" ; then
 fi
 
 
+# Building on Travis
+if [ "${TRAVIS}" = "true" ] ; then
+    DISABLE_REVVIDEOGRABBER=-Denable_revvideograbber=0
+fi 
```

Comment:
As mentioned previously, I'm not sure it's a good idea to give the build tools explicit awareness of Travis, but I'm not sure how to deal with this one in a nice way.